yt_dlp/extractor/tiktok.py

```diff
@@ -911,3 +911,47 @@ def _real_extract(self, url):
         if self.suitable(new_url):  # Prevent infinite loop in case redirect fails
             raise UnsupportedError(new_url)
         return self.url_result(new_url)
+
+
+class TikTokLiveIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?tiktok\.com/@(?P<id>[\w\.-]+)/live'
+    IE_NAME = 'tiktok:live'
+
+    def _real_extract(self, url):
+        uploader_id = self._match_id(url)
+        webpage = self._download_webpage(url, uploader_id, headers={'User-Agent': 'User-Agent:Mozilla/5.0'})
```

Comment:
This `User-Agent` workaround may not work for all users (possibly a fingerprinting or regional thing). 
It's fine for now, but looking ahead, do you know of any other source for the `room_id` value besides the webpage?

Reply:
After checking out other githubs about `TikTok`, it seems that currently the only way to get the `room_id` is from the webpage